### PR TITLE
Add update translation files GitHub action

### PR DESCRIPTION
**Pull request by MauricioFauth**
Related to https://github.com/phpmyadmin/scripts/pull/44.

### CODE REVIEW

File: .github/workflows/update-po.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            composer-${{ runner.os }}-${{ matrix.php-version }}-
+
+      - name: Install dependencies
+        run: |
+          composer update --no-interaction --no-progress
+
+      - name: Setup git config
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
```
Comment (williamdes) [inline .github/workflows/update-po.yml:50]:
```suggestion
          git config user.name github-actions
          git config user.email github-actions@github.com
```
would be better to use our bot as a committer `phpMyAdmin bot <bot@phpmyadmin.net>`

**Review by williamdes — APPROVED** [review]

**Comment by mauriciofauth** [comment]
It worked as expected! :100:

How to manually trigger the workflow:
- Go to: https://github.com/phpmyadmin/phpmyadmin/actions/workflows/update-po.yml
- Click on the _Run workflow_ button.
- See commit like: https://github.com/phpmyadmin/phpmyadmin/commit/7d8b9a48a9b432a034f4f2f08c13345a16899263

**Comment by mauriciofauth** [comment]
Hi, @nijel. Could you help with something?

How is the better way to use the Weblate's `wlc` in this GitHub Action?
We have to pass a Weblate API key to the `--key` option, right? For example:
```
wlc --key ${{ secrets.WEBLATE_API_KEY }} --url https://hosted.weblate.org/api/ lock
```
On Weblate, I only found my personal API key, should we use it or should we use a bot's "personal" API key?

**Comment by nijel** [comment]
Project API tokens are planned, but not yet implemented, see https://github.com/WeblateOrg/weblate/issues/4974. For not I'd recommend to create a bot user for that.

**Pull request merged**